docs/api-docs/cart-and-checkout/embedded-checkout-tutorial.md

```diff
@@ -175,18 +175,20 @@ Next, generate cart redirect URLs by sending a `POST` request to `/carts/{{cart_
 ```
 
 ### Redirecting a logged-in customer to embedded checkout
+
 For some use cases, you may want your customer to log in before they can begin the checkout process.
 
-Customers can log in using the [Customer Login API](https://developer.bigcommerce.com/api-docs/customers/customer-login-api#logging-in-a-customer). 
+Customers can log in using the [Customer Login API](https://developer.bigcommerce.com/api-docs/customers/customer-login-api#logging-in-a-customer).
```

Comment:
Customers can log in using the [Customer Login API](https://developer.bigcommerce.com/api-docs/storefront/customer-login-api).